openvino_tensorflow/ovtf_builder.cc

```diff
@@ -4017,13 +4017,6 @@ Status Builder::TranslateGraph(
     passes.run_passes(ng_function);
   }
   OVTF_VLOG(5) << "Done with passes";
-  //
-  // Request row-major layout on results.
-  //
-  for (auto result : ng_function->get_results()) {
-    result->set_needs_default_layout(true);
```

Comment:
Why are we removing this?

Reply:
According to this, there's no implementation for this method anymore: https://github.com/openvinotoolkit/openvino/blob/a79830cb55623ce2654f0685b0ca51b3f412195a/src/core/include/openvino/op/result.hpp#L34

Earlier for 2021.4.2, it used to be under ngraph headers https://github.com/openvinotoolkit/openvino/blob/9424272cce8f1169c779d21a4384ac8a73e8b0c2/ngraph/core/include/ngraph/op/result.hpp#L35